messages/src/lib.rs

```diff
@@ -2077,9 +2078,63 @@ impl<Q: QueryType> Decode for AggregationJobInitializeReq<Q> {
     }
 }
 
+/// DAP protocol message representing the round of an aggregation job.
```

Comment:
```suggestion
/// Type representing the round of an aggregation job.
```

DAP doesn't define an `AggregationJobRound` message or structure, instead using a raw `u16` field. (We could also drop this type entirely, though I'm OK keeping it as it's nice to have type-safe numeric types.)

Reply:
That's a fair hair to split. I prefer keeping `AggregationJobRound` for clarity.